indirect.h

```diff
@@ -351,26 +351,26 @@ struct std::hash<xyz::indirect<T, Alloc>> {
 
 namespace xyz {
 
-template <class T>
-concept is_formattable = requires(T t) { std::formatter<T, char>{}; };
-    
-}
+template <class T, class charT>
+concept is_formattable = requires(T t) { std::formatter<T, charT>{}; };
+
+}  // namespace xyz
 
 template <class T, class Alloc, class charT>
-  requires xyz::is_formattable<T>
-struct std::formatter<xyz::indirect<T, Alloc>, charT> : std::formatter<T> {
-  constexpr auto parse(format_parse_context& ctx)
-      -> format_parse_context::iterator {
+  requires xyz::is_formattable<T, charT>
+struct std::formatter<xyz::indirect<T, Alloc>, charT> : std::formatter<T, charT> {
+  template <class ParseContext>
+  constexpr auto parse(ParseContext& ctx) -> typename ParseContext::iterator {
     return std::formatter<T>::parse(ctx);
```

Comment:
I am sorry, I am not very careful, it's weekend 🙂 
There are two more places, where `std::formatter<T>` needs to be replaced by `std::formatter<T, charT>`. Here and..